spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -7773,6 +7773,21 @@ class KotlinExampleProperties {
 }
 ----
 
+In order to allow <<appendix-configuration-metadata#configuration-metadata-annotation-processor,
+generating your own metadata by using the annotation processor>> with Gradle, the
+`spring-boot-configuration-processor` dependency should be declared in the `kapt`
+configuration, as shown in the following example:
+
+[source,groovy,indent=0,subs="verbatim,quotes,attributes"]
+----
+	dependencies {
+		kapt "org.springframework.boot:spring-boot-configuration-processor"
+	}
+---- 
```

Comment:
Is there a reason to focus on Gradle only here? I know there is a link below but if both are handled in that link I'd rather just mention that.

Reply:
Yes. If you have a link that points to what needs to be done, I see no reason to copy/paste instructions for one build system. This is also the only gradle snippet in that section.